src/main/scala/com/codacy/client/bitbucket/client/BitbucketClientBase.scala

```diff
@@ -9,6 +9,7 @@ import com.codacy.client.bitbucket.client.Authentication._
 import com.codacy.client.bitbucket.util.HTTPStatusCodes
 import com.codacy.client.bitbucket.util.Implicits.URIQueryParam
 import play.api.libs.json._
+import play.api.libs.ws.WSClient
```

Comment:
We have the import com.codacy.client.bitbucket.WSWrapper.WSClient that will give the correct version